src/Cli/dotnet/commands/dotnet-package/PackageCommandParser.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+
+namespace Microsoft.DotNet.Cli
+{
+    internal class PackageCommandParser
+    {
+        public static readonly string DocsLink = "";
```

Comment:
will there eventually be documentation? If so, I'd create the aka.ms link today and include it as a placeholder